docs/sources/configure/options.md

```diff
@@ -502,6 +502,31 @@ attributes:
       exclude: ["k8s.pod.*"]
 ```
 
+Additionally, you can use "`*`" wildcards as metric names to add and exclude attributes for
+groups of metrics having the same name. For example:
+
+```yaml
+attributes:
+  select:
+    http_*:
+      include: ["*"]
+      exclude: ["http_path", "http_route"]
+    http_client_*:
+      # override http_* exclusion
+      include: ["http_path"]
+    http_server_*:
+      # override http_* exclusion
+      include: ["http_route"]
+```
+
+In the previous example, all the metrics with a name starting with `http_` (or `http.`) would include all
+the possible attributes but `http_path` and `http_route` (or `http.path`/`http.route`).
+But the `http_client_*` and `http_server_*` sections would override the base configuration, enabling the
```

Comment:
Could we add a second example for the commented out code for this scenario?

Reply:
Sorry, which commented out code do you refer?

Reply:
I see it is a comment explaining an override.